composer/cicd_sample/utils/add_dags_to_composer.py

```diff
@@ -39,7 +39,9 @@ def _create_dags_list(dags_directory: str) -> Tuple[str, List[str]]:
     return (temp_dir, dags)
 
 
-def upload_dags_to_composer(dags_directory: str, bucket_name: str) -> None:
+def upload_dags_to_composer(
+    dags_directory: str, bucket_name: str, name_replacement: str = "dags/"
```

Comment:
Can we simply do `name_replacement: str = "dags"` and use it to replace with

```
dag = dag.replace(f"{temp_dir}", name_replacement)
```
I worry that having to keep track of the extra `/` might bite folks in the future. But, if this is necessary because you don't want to have to replace all of `{temp_dir}` accidentally with the wrong stuff then it's fine as is.

Reply:
Would it be better if there was more of an explanation for the parameter? Just to reduce the chance of being burned?